### ISSUE TITLE
Validation set ratio

### ISSUE THREAD

**Issue by longkhanh-fam** [yochaiye/LipVoicer#6]
Hi,
Which subsets do you use to validate in lrs2 and lrs3, only val subset or both val and pretrain? And what is the used ratio between subsets.

Thank you for your answer

**Comment by yochaiye** [comment]
For LRS3 I used pretrain+trainval for training, and pretrain+train for LRS2.
I didn't run any validation during training because it is very time consuming.

**Comment by longkhanh-fam** [comment]
Thank you ! I will close the issue here
